drivers/wifi/nrf700x/src/fmac_main.c

```diff
@@ -324,12 +341,17 @@ int nrf_wifi_reg_domain(const struct device *dev, struct wifi_reg_domain *reg_do
 			goto err;
 		}
 	} else if (reg_domain->oper == WIFI_MGMT_GET) {
+
+		rpu_ctx_zep->reg_chnl_info = reg_domain->chnl_info;
```

Comment:
I guess a check is necessary to ensure that `reg_domain->chnl_info` is valid ?

Reply:
chnl_info is static memory, so no need to check.